packages/netsuite-adapter/src/filters/element_references.ts

```diff
@@ -135,9 +135,11 @@ const getServiceElemIDsFromPaths = (
 ): ElemID[] =>
   foundReferences
     .flatMap(ref => {
-      const absolutePath = pathPrefixRegex.test(ref)
-        ? resolveRelativePath(element.value[PATH], ref)
-        : FILE_CABINET_PATH_SEPARATOR.concat(ref)
+      const absolutePath = resolveRelativePath(element.value[PATH], ref)
+      // TODO: The log should be removed when SALTO-4025 is communicated.
+      if (!pathPrefixRegex.test(ref)) {
+        log.debug('Found a possible file reference without a path prefix: %s', ref)
+      }
       return [ref, absolutePath].concat(
```

Comment:
also for now we should keep the original behavior from [here](https://github.com/salto-io/salto/commit/a05213074a6569dc4ebe217abfdd5e3094d69be8)-
```suggestion
      return [absolutePath].concat(
```